scripts/scene/EditingManager.gd

```diff
@@ -98,6 +93,10 @@ func _on_editing_state_input(event: InputEvent) -> void:
 	if event is InputEventKey:
 		if event.pressed and event.keycode == KEY_ESCAPE:
 			state_manager.send_event(Events[StateEvent.EDITING_STOP])
+	elif event is InputEventMouseButton:
+		if event.pressed:
+			match event.button_index:
+				1: room_editor.handle_select_input(event, camera.offset, camera.zoom);
 	
```

Comment:
_on_editing_state_input does not seem to trigger now. Shall we implement the same 'tooltip' for the room with basic info (capacity, resources etc). You mentioned adding areas to the rooms. Is this what you planned to do, @conalli ?

Reply:
It was not possible to do both the assigning and editing so I just temporarily stopped the room edit for now, both room edit and assigning will need to be updated so that they can work together but we need to decide how to do that

Reply:
Understood. Sorry, if you told me that before and I forgot.